src/App.scss

```diff
@@ -89,4 +86,19 @@ p {
     margin-top: 0px;
     width: 100%;
     overflow: hidden;
+}
+
+.root { 
+}
+
+header {
+  position: absolute;
+  width: 100%;
+  top: 0;
+  left: 0;
+  z-index: 10000;
+}
+
+main {
```

Comment:
It looks like this gets used in `Home.jsx` as `home-main`. That's cool! Why does it work?